frontend/bank-transfer-ui/src/app/main/account-details/account-details.component.css

```diff
@@ -0,0 +1,18 @@
+.cards {
+  display: flex;
+}
+.cards .card:nth-child(1){
+  flex: .25;
+}
+h5, p {
```

Comment:
To powinno polecieć do typograficznych styli globalnych, a nie być zagnieżdżone w account-details